tree/tree/src/TChain.cxx

```diff
@@ -2644,13 +2645,27 @@ void TChain::SetDirectory(TDirectory* dir)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Set the input entry list (processing the entries of the chain will then be
-/// limited to the entries in the list).
+/// \brief Set the input entry list (processing the entries of the chain will
+///        then be limited to the entries in the list).
+///
+/// \param[in] elist The entry list to be assigned to this chain.
+/// \param[in] opt An option string. Possible values are:
+///   - "" (default): both the file names of the chain elements and the file
+///     names of the TEntryList sublists are expanded to full path name.
+///   - "ne": the file names are taken as they are and not expanded
+///   - "sync": the function assumes the argument \p elist has exactly one sub
+///     entry list per file in the chain. Each sub entry list must have exactly
+///     the same treename and filename as the corresponding tree in the chain.
```

Comment:
describe effects, not pre-requisites:

```suggestion
///   - "sync": the TChain will go through the TEntryList in lockstep with the
        trees in the chain rather than performing a lookup based on
        treename and filename. This is mostly useful when the TEntryList
        has multiple sublists for the same tree and filename. 
```